src/public/publicAPIs.ts

```diff
@@ -59,9 +62,14 @@ export function initialize(callback?: () => void, validMessageOrigins?: string[]
       // of the parent window, and this message contains no data that could pose a security risk.
       GlobalVars.parentOrigin = '*';
       const messageId = sendMessageRequestToParent('initialize', [version]);
-      GlobalVars.callbacks[messageId] = (context: string, clientType: string) => {
+      GlobalVars.callbacks[messageId] = (context: string, clientType: string, props: {}) => {
```

Comment:
Made the suggested change.

**Reason for introducing props:** I noticed the parameters in the callback function were getting increased, so a dictionary seemed optimal choice, hence introduced props so that any return value could be put in that dictionary.